server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -85,48 +87,64 @@ public Content render(
                                     programs.getActiveProgramDefinition(name),
                                     programs.getDraftProgramDefinition(name),
                                     request,
-                                    profile))))
-            .with(renderDownloadExportCsvButton());
+                                    profile))));
 
     HtmlBundle htmlBundle =
         layout
             .getBundle()
             .setTitle(pageTitle)
             .addMainContent(contentDiv)
-            .addModals(publishAllModal)
             .addFooterScripts(layout.viewUtils.makeLocalJsTag("admin_programs"));
+    if (maybePublishModal.isPresent()) {
+      htmlBundle = htmlBundle.addModals(maybePublishModal.get());
+    }
```

Comment:
suggestion (non-blocking): no need to reassign the variables, addModals mutates
```suggestion
    maybePublishModal.ifPresent(htmlBundle::addModals);
```